test/suite/helper.py

```diff
@@ -115,11 +115,35 @@ def copy_wiredtiger_home(olddir, newdir, aligned=True):
             "WiredTigerPreplog" not in fullname:
             # Use a dd command that does not align on a block boundary.
             if aligned:
-                shutil.copy(fullname, newdir)
+                try:
+                    shutil.copy(fullname, newdir)
+                except OSError as e:
+                    # If the copy failed it means the file was removed underneath us in the
+                    # short time after we verified the files existence. This can happen if we
+                    # are copying wiredtiger log files at the same time they are getting
+                    # archived by the background log server thread. To avoid failing in this case
+                    # we can log/print the removed file and continue.
+                    if "WiredTigerLog" not in fullname:
+                        raise e
+                    else:
+                        self.printVerbose(2, 'Skipping logfile %s: No longer exists' % fname)
```

Comment:
Perhaps instead of a verbose a `self.pr()` call would be useful so that it ends up in `results.txt` and doesn't depend on someone setting a verbose level. It seems reasonably useful to print out all the time it happens to that file.

Reply:
Good point, now fixed